rotkehlchen/api/rest.py

```diff
@@ -2882,6 +2885,10 @@ def decode_evm_transactions(
         else:  # no break in the for loop, success
             result = True
 
+        for evm_chain, refresh_balances in refresh_balances_dict.items():
+            self.rotkehlchen.chains_aggregator.get_evm_manager(evm_chain).transactions_decoder.post_process(
+                refresh_balances=refresh_balances,
+            )
```

Comment:
```suggestion
            chain_manager.transactions_decoder.post_process(
                refresh_balances=refresh_balances,
            )
```

Reply:
No, this won't work. It will take the last used chain aggregator, and it will not respect the evm chain